pkg/resources/grant_ownership_acceptance_test.go

```diff
@@ -884,6 +884,7 @@ resource "snowflake_grant_account_role" "test" {
 }
 `, accountRoleName, databaseName, userName)
 
+	// TODO: build these configs from builders
```

Comment:
next pr?

Reply:
Nope, it was added to readme as:
```
add provider to resource/datasource models (use in the grant_ownership_acceptance_test)
```
I missed this one (i.e. adding SNOW-1501905). Added.